docs/contributing.rst

```diff
@@ -23,12 +23,15 @@ Discussion forums
 Conversation about *sgkit* happens in the following places:
 
 1.  `GitHub Issue Tracker`_: for discussions around new features or bugs
-2.  `Python for Statistical Genetics forum`_: for general discussion
+2.  `GitHub Discussions`_: for general discussion
```

Comment:
I think it's nice to point people here for general support stuff, otherwise they'll use the issues. something like:

> General discussions and questions like "how do I do X" or "what's the best way to do Y"?

Reply:
Fixed